db/migrate/20180120042303_create_expenses.rb

```diff
@@ -3,7 +3,7 @@ def change
     create_table :expenses do |t|
       t.string :name, null: false
       t.date :due_date
-      t.string :frequency, null: false
+      t.integer :frequency, null: false
```

Comment:
Does it make sense to set a default value?

Reply:
It does make sense, thanks for the pick up, done in sha: 130cd96